src/Utils/CloudResponse.php

```diff
@@ -84,9 +84,9 @@ public function __construct($header, $response, $url, $parameters) {
         preg_match_all("/^X-Resultset-([^:]+): ([\d]+).+$/m", $header, $p);
         $header_fields = array_combine($p[1], $p[2]);
 
-        $this->total = $header_fields['Total'] ?: -1;
-        $this->page_limit = $header_fields['Limit'] ?: -1;
-        $this->page = $header_fields['Page'] ?: -1;
+        $this->total = $header_fields['Total'] ?? -1;
+        $this->page_limit = $header_fields['Limit'] ?? -1;
+        $this->page = $header_fields['Page'] ?? -1;
```

Comment:
The null-coalesce operator (`??`) is only available in PHP >= 7.0. Since this library supports PHP 5.4+, we can't merge this pull request. 

Instead, you can write this out the long way: 

```php
$this->total = isset($header_fields['Total']) ? $header_fields['Total'] : -1;
```